lib/Test/Stream/Formatter/TAP.pm

```diff
@@ -99,6 +116,193 @@ sub _autoflush {
     select $old_fh;
 }
 
+sub event_tap {
+    my $self = shift;
+    my ($e, $num) = @_;
+
+    my $converter = $CONVERTERS{ref($e)};
+
+    $num = undef if $self->{+NO_NUMBERS};
+
+    # Legacy Support for $e->to_tap
+    unless ($converter) {
+        my $legacy = $e->can('to_tap') or return;
+        return if $legacy == \&Test::Stream::Event::to_tap;
```

Comment:
It is deprecated, and will be removed as soon as it is safe. I put in a reasonable level of safety.